### ISSUE TITLE
Vaquita genetic map snag

### ISSUE THREAD

**Issue by silastittes** [popsim-consortium/analysis2#111]
The pipeline as written seems to require a genetic map at various places. 
I'm getting the following error [here](https://github.com/popsim-consortium/analysis2/blob/ae85199ff11eb2d052dd55eae1544675642bedad/workflows/n_t.snake#L488) :

```
ERROR:snakemake.logging:RuleException:
ValueError in file analysis2/workflows/n_t.snake, line 488:
GeneticMap 'PhoSin/None' not in catalog ()
```

I think the issue is most salient in this [function](https://github.com/popsim-consortium/analysis2/blob/ae85199ff11eb2d052dd55eae1544675642bedad/workflows/gone.py#L50), where various aspects of the code depend on [GeneticMap](https://github.com/popsim-consortium/stdpopsim/blob/a93b21179822f47f86d673a9932803e9881fd9c6/stdpopsim/genetic_maps.py#L13) methods.

I'm wondering if folks more familiar with stdpopsim can help me navigate this.
Maybe a silly solution, but maybe an update to the catalog code that makes a constant genetic map file that can be downloaded so that the GeneticMap methods are still available? Maybe this is already the case and I just need to adjust how to initial the pipeline? 
Thanks for thoughts!

**Comment by andrewkern** [comment]
tagging @petrelharp here -- is there no PhoSin map?

**Comment by petrelharp** [comment]
No, there is no genetic map (perhaps unsurprisingly?)
![Screenshot from 2024-05-09 17-14-48](https://github.com/popsim-consortium/analysis2/assets/1046249/74497e02-086c-4264-8e1e-aac9fb8a2ee2)

Sounds like the pipeline needs to be modified - there is no reason we need to use a map?

**Comment by silastittes** [comment]
I don’t think we need one for the aims of the manuscript. Could we make a “fake” constant one for Vaquita for this project <ducks>? My initial impression is it will take a lot of refactoring to make the pipeline work without one, but that may reflect my lack of familiarity with the code bases.

**Comment by petrelharp** [comment]
Well, probably? That sounds like the long way round but then I'm not familiar with the pipeline. The [error you got](https://github.com/popsim-consortium/analysis2/blob/main/workflows/n_t.snake#L488) could be fixed relatively easily with some special-case logic, but I haven't seen how often this comes up?

**Comment by silastittes** [comment]
Yeah, it may be a smaller issue than I initially thought. I’ll try making some minor edits and see if I can get it working without a genetic map.

**Comment by silastittes** [comment]
Ok, I think I got this sorted out -- PR forthcoming. MSMC is failing to converge, but maybe that won't be an issue for the production pipeline?

**Comment by andrewkern** [comment]
guessing MSMC isn't converging in the tiny config cause of too few sites?

**Comment by silastittes** [comment]
I think so too. Giving both species on more clean run with tiny config to make sure everything is square then I’ll PR the update.

**Comment by silastittes** [comment]
fixed with #112
